tests/autofix_files/async910.py.diff

```diff
@@ -205,8 +205,16 @@
 
 
  async def foo_comprehension_3():
-@@ x,3 x,4 @@
+@@ x,6 x,7 @@
 
  async def await_everywhere_except_gen_target():  # error: 0, "exit", Statement("function definition", lineno)
      (await x async for x in bar())
 +    await trio.lowlevel.checkpoint()
+
+
+ # Issue #226
+@@ x,3 x,4 @@
+                 pass
+     except Exception:
+         pass
++    await trio.lowlevel.checkpoint()
```

Comment:
once again this looks kinda weird in the github diff view, due to viewing a syntax-highlighted diff file in diff view. But the actual diff is entirely as expected.